docker/playground/.env

```diff
@@ -28,3 +28,5 @@ SPARK_VERSION=3.4.2
 SPARK_BINARY_VERSION=3.4
 SPARK_HADOOP_VERSION=3.3.4
 ZOOKEEPER_VERSION=3.6.3
+PROMETHEUS_VERSION=v2.45.2
```

Comment:
why leave a leading `v`?

Reply:
prometheus docker image requires.

See more tags in https://hub.docker.com/r/prom/prometheus/tags

Reply:
that's tag, not version. if wanted, we may use `apache-kyuubi:${KYUUBI_VERSION}-spark` or something